packages/dev/docs/src/docs.css

```diff
@@ -506,6 +506,7 @@ footer li:last-child:after {
 
 .article pre {
   background: var(--spectrum-global-color-gray-75);
+  border: 1px solid var(--spectrum-gray-200);
```

Comment:
Added a very subtle border around code examples for a bit more structure as well.

Reply:
300 seemed too much... 😂

Reply:
:-/ dunno then
can live with it for now, we'll see if anyone complains